force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +314,28 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer stringLocation=-1;
+        
+        if (words.contains(wordToFind)){
```

Comment:
I like that you checked here for contains(), but also indexOf() returns -1 if it is not found, so you could shorten this even more by just declaring `String stringLocation = words.indexOf(wordToFind);`